components/icon/Icon.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Author: bladimirardiles
+ * Component File Name: icon.js
+ * Component Name: icon
+ * Project: design-system-component-library-football-fanatics
+ * Date: Mon 16/10/2023 - 13:14
+ */
+
+import React, { useMemo } from 'react';
+import './icon.css';
+import { EIconColors, EIconSizes, EIConTypes } from './icon.enum';
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  ArrowDownIcon,
+  ArrowUpIcon,
+} from '@heroicons/react/24/solid';
+
+interface IProps {
```

Comment:
Plz extract Interface to its own file as it was done for the button

Reply:
done!